feature/eks-cluster-creation/README.md

```diff
@@ -0,0 +1,48 @@
+## Terraform scripts to provision EKS cluster #
+***
+
+
+The Amazon Elastic Kubernetes Service (EKS) is the AWS service for deploying, managing, and scaling containerized applications with Kubernetes. This repo contain terraform files, to automate the EKS cluster deployment.
+***
+
+#### Requirements:
+- AWS account with the IAM permissions listed on the [EKS module documentation](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/iam-permissions.md)
+- Configured AWS CLI
+- AWS IAM Authenticator
```

Comment:
do I really need that for terraform?

Reply:
Not anymore, as we don't use aws CLI anymore. Kubeconfig applied automatically here: https://github.com/mum0l/epiphany/blob/1442-EKS_cluster_setup/feature/eks-cluster-creation/aws-eks.tf#L35 so each time cluster is build, we're already connected into.